client/queries/eventQuery.ts

```diff
@@ -8,8 +8,12 @@ export class Events extends AccountQuery {
   }
 
   constructor(connection: Connection) {
-    super(connection, Event, new Map([
-        ["authority",  new PublicKeyCriterion(8)],
+
+    super(connection, Event, new Map<string, PublicKeyCriterion | BooleanCriterion>([
```

Comment:
```
new Map<string, Criterion<unknown>>([
```
might work better here as an example for the next one we do. I'm not sure I see an advantage is being more specific than the values needing to be any type of `Criterion`.

Reply:
Yeah :/ I went back and forth a few times, I'm fine with `unknown`

Reply:
at least the linter doesn't give us attitude about `unknown` like it does with `any` 😌